resources/aws_ec2.go

```diff
@@ -547,9 +570,16 @@ func (obj *AwsEc2Res) snsWatch() error {
 			if err := msg.err; err != nil {
 				return err
 			}
-			log.Printf("%s: State: %s", obj, msg.str)
-			obj.StateOK(false)
-			send = true
+			if msg.run {
```

Comment:
This is a fine approach, but here's an alternate idea:

We currently have random string messages (which are all internal to us) -- Instead, please make a bunch of constant iota's and use those instead. This can be one as well. In fact, the first commit could be to switch all the existing code to use iota of uint8 or something.